tests/outputs/TestDiagram.cpp

```diff
@@ -134,6 +134,6 @@ TEST(Diagram, writeEmpty) {
     std::ifstream stream(filePath);
     ASSERT_TRUE(stream.fail()) << "The Diagram file has been written to even though there are no generators which model requires it." << std::endl;
   } catch (std::exception& e) {
-    ASSERT_TRUE(false) << "An IO error has occured" << std::endl;
+    ASSERT_TRUE(false) << "An IO error has occured\n" << std::endl;
```

Comment:
why add another end of line character ?

Reply:
Ah gautier induced me in error, see his remark above

Reply:
newline at the end of the file not here